Lesson3/IFileHandling.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Project_Console
+{
+    public interface IFileHandling
+    {
+        void DisplayData(List<string> parsedData);
```

Comment:
`Scope`: Should the class be responsible for showing data?

Reply:
after some thought not really. how and when to display the data is up to the developer and in this case belongs in the Program.cs code.